src/pages/data-visualization/getting-started/other-frameworks.mdx

```diff
@@ -1,13 +1,19 @@
 ---
 title: Get started
-description: The Carbon Design System supports vanilla JS, React and Angular as core parts of the product. But you can still build components even if you're using a different framework.
+description: The Carbon Design System supports Vanilla JS, React and Angular as core parts of the product. But you can still build components even if you're using a different framework.
```

Comment:
Vanilla isn't a proper noun here. It's not the name of our js framework, just an adjective meaning 'regular' or 'normal'.

Reply:
@vpicone las time that I looked this up it looks like the convention is that it is treated as a proper noun.

Reply:
@connor-leech you might have seen that on http://vanilla-js.com/ which is a joke site promoting just using regular javascript instead of X framework.

Reply:
I noticed some folks on Stack Exchange and the like also capitalized it, but I know no higher authority on JS than Vince Picone, so sounds good to me.

Reply:
This is a subjective matter, there's no "correct" way of saying vanilla (as a noun, capital, lowercase etc.) as there's no authority or organization that standardizes the use of the word.

I would suggest keeping things consistent, "vanilla" is uppercased in most places on the site:
![image](https://user-images.githubusercontent.com/14989804/66610435-8a9b6f00-eb89-11e9-9aa6-5b44f270d40b.png)

Reply:
I don't really mind either way, but the highlighted examples are capitalized because they are headers, so any word in that position would be capitalized. We can leave it lower case for now and make a style decision when Jan gets back into town.

Reply:
You're right, I think these 2 instances though are the only cases where vanilla isn't the first word in the header

![image](https://user-images.githubusercontent.com/14989804/66616518-179cf300-eb9f-11e9-9a2b-5934a1f9790b.png)